src/shared/views/RecommendedCards.js

```diff
@@ -164,6 +178,15 @@ class _RecommendedCards extends React.Component<Props> {
         );
     };
 
+    getTeamName = () => {
+        // request the team display name from firebase
+        // the result is then available under state.firebase.data.teamDetails
+        const { firebase, teamId } = this.props;
+        if (teamId) {
+            firebase.watchEvent('once', `v2/teams/${teamId}`, 'teamDetails');
```

Comment:
there is no `teamDetails` endpoint in firebase. I'm not understanding what you want to do here. I might be a bit confused and you might set the `state.firebase.data.teamDetails` endpoint somewhere else, but where?

Reply:
`teamDetails` is where I tell the firebase client to store the data it downloads from the backend. This is only a local value, inside the app's data storage. The line above says "fetch `v2/teams/${teamId}` only once and place the result in the local firebase tree under `..../teamDetails/`. Then it's available from anywhere in the app under that same location (which is your other question above).